alibi_detect/cd/classifier.py

```diff
@@ -188,41 +182,60 @@ def score(self, X: Union[np.ndarray, list]) -> float:
             splits = self.skf.split(x, y)
 
         # iterate over folds: train a new model for each fold and make out-of-fold (oof) predictions
-        preds_oof, idx_oof = [], []
+        preds_oof_list, idx_oof_list = [], []
         for idx_tr, idx_te in splits:
             x_tr, y_tr, x_te = x[idx_tr], np.eye(2)[y[idx_tr]], x[idx_te]
             clf = tf.keras.models.clone_model(self.model)
             clf.compile(**self.compile_kwargs)
             clf.fit(x=x_tr, y=y_tr, **self.fit_kwargs)
             preds = clf.predict(x_te, batch_size=self.fit_kwargs['batch_size'])
-            preds_oof.append(preds)
-            idx_oof.append(idx_te)
-        preds_oof = np.concatenate(preds_oof, axis=0)[:, 1]
-        idx_oof = np.concatenate(idx_oof, axis=0)
-        drift_metric = self.metric_fn(y[idx_oof], preds_oof)
-        return drift_metric
-
-    def predict(self, X: Union[np.ndarray, list], return_metric: bool = True) \
-            -> Dict[Dict[str, str], Dict[str, Union[int, float]]]:
+            preds_oof_list.append(preds)
+            idx_oof_list.append(idx_te)
+        preds_oof = np.array(np.concatenate(preds_oof_list, axis=0)[:, 1])
+        idx_oof = np.array(np.concatenate(idx_oof_list, axis=0))
```

Comment:
Would replace `np.array` with `np.asarray` as the latter will not make a copy if possible, e.g. https://stackoverflow.com/questions/14415741/what-is-the-difference-between-numpys-array-and-asarray-functions

Reply:
Oh yeah I think that can be removed now actually as I took another route to satisfy mypy.